include/odp/api/spec/std_types.h

```diff
@@ -40,6 +41,15 @@ extern "C" {
  */
 typedef uint32_t odp_percent_t;
 
+/** Unaligned uint16_t type */
+typedef uint16_t ODP_ALIGNED(1) odp_una_u16_t;
```

Comment:
Looking at GCC documentation it seems to me that in this kind of typedef the type attribute should be at the end, i.e.: typedef uint16_t odp_una_u16_t ODP_ALIGNED(1); With structs and unions it is different.

Reply:
fixed in v2